playbooks/templates/standalone_parameters.yaml.j2

```
@@ -159,10 +159,12 @@ parameter_defaults:
     # for cephadm support on OSP 17
     tripleo_cephadm_ceph_nfs_bind_addr: "{{ public_api }}"
 {% endif %}
+{% if standalone_extra_config|length > 0
```

Did you forget the closing sign?
```suggestion
{% if standalone_extra_config|length > 0 %}
```